app/models/healthcheck.rb

```diff
@@ -32,6 +33,16 @@ def read_revision_file
     nil
   end
 
+  def all_healthy?
```

Comment:
This will result in all the health checks getting run twice?

how about something like

```
checks = {
  sidekiq: sidekiq_healthy?
}

checks.merge(healthy: check.values.all?)
```

Reply:
Also is failing the specs? Looks like it might not be returning service unavailable?

Reply:
I had the same though, yeah. Will fix